tortoise/tortoise.go

```diff
@@ -943,7 +955,7 @@ func (t *turtle) computeLocalOpinion(ctx *tcontext, lid types.LayerID) (map[type
 		return opinion, nil
 	}
 
-	weight, err := computeExpectedVoteWeight(t.atxdb, t.epochWeight, lid, t.Last)
+	weight, err := computeExpectedVoteWeight(t.atxdb, t.epochWeight, t.Processed.Sub(1), t.Processed)
```

Comment:
submit separately

Reply:
i don't understand this change. it should be the number of layers that could've cast votes on lid, no?

Reply:
ah. this is the local opinion. ok. i see in the other PR